tpm.c

```diff
@@ -58,51 +58,29 @@ struct tpm *enable_tpm(void)
 {
 	struct tpm *t = &tpm;
 
+	/* TODO: ACPI TPM discovery */
+
 	find_interface_and_family(t);
 
 	switch (t->intf) {
-	case TPM_DEVNODE:
```

Comment:
I thought you were going to leave DEVNODE and UEFI in tpmlib and strip them from the kernel versions?

Reply:
Nope, stripping them completely for now. Will bring them back along with a working implementation instead of the stubs.